### ISSUE TITLE
Add Energy Cell as an ammo subtype

### ISSUE THREAD

**Issue by etiquettestartshere** [foundryvtt/dnd5e#4788]
https://github.com/foundryvtt/foundryvtt-premium-content/issues/873 to unblock this issue for the dmg module.

This may have to wait until the new SRD(?)

**Comment by arbron** [comment]
Ammo subtypes don't require base items, so it should be entirely possible to add this without the new SRD.

**Comment by etiquettestartshere** [comment]
I'm not sure if that will be enough for them to unblock it or not--their concern seems to be locking use of the futuristic ammo type behind a paywall (understandable). Though maybe something could be worked out with them adding their version to `DND5E.ammoIds` now and the SRD Energy Cell, if it exists, redirecting (or the other way around). I guess we can wait and see, I'll comment on that issue again when 4.2 drops and if it is not enough, make another issue here about `DND5E.ammoIds`.

**Comment by arbron** [comment]
At this point, `DND5E.ammoIds` is not even used by the system. It was added along with `containerTypes` and `focusTypes` for the starting equipment system, but only `focusTypes` ended up actually being used by that system.
